### DISCUSSION TITLE
Error : UserWarning: Facebook language detected as ru_RU - for best results, set to en_US   warnings.warn(

### DISCUSSION THREAD

**Discussion by jurgenmecaj** [kevinzg/facebook-scraper#985] [Q&A]
i have this error , please help me . 
UserWarning: Facebook language detected as ru_RU - for best results, set to en_US
  warnings.warn(

**Comment by DiegoChito** [comment]
I solved it. You have to change your language in the facebook settings from ru_RU to en_US then you have to get the cookies from that language.
